Source/CLI/rawcooked.1

```diff
@@ -158,6 +158,10 @@ Do not carry out coherency check (see above).
 .B --conch
 Conformance check of the format, effective only when format is supported.
 .br
+Currently partially implemented for DPX.
+.br
+Disable encoding and decoding.
+.br
 This is currently partially implemented for DPX.
```

Comment:
Then this line should ne cancelled, I presume?

Reply:
> Then this line should be cancelled, I presume?

Good catch, I removed the added so duplicated content.